devops/README.md

```diff
@@ -1,17 +1,211 @@
+[![insiders-fast](https://github.com/Azure/azure-osconfig/actions/workflows/insiders-fast.yml/badge.svg)](https://github.com/Azure/azure-osconfig/actions/workflows/insiders-fast.yml) | The latest and greatest features and bug fixes. This channel is updated frequently.
+[![prod](https://github.com/Azure/azure-osconfig/actions/workflows/prod.yml/badge.svg)](https://github.com/Azure/azure-osconfig/actions/workflows/prod.yml) | The stable channel. This channel is updated less frequently and is recommended for production use.
+
+*If you see any missing platforms or architectures that would help, please contact osconfigcore@microsoft.com.
+
+| Platform | Architecture | Package Repository | Package Removal
+-----|-----|-----|-----
+Amazon Linux 2 | x86_64 | [insiders-fast](#amazon-linux-9---insiders-fast-amd64) | [link](#amazon-linux-9---package-removal)
+Azure Linux 2 (mariner-2) | x86_64 | [insiders-fast](#azure-linux-2-cbl-mariner2-mariner2---insiders-fast-amd64) | [link](#azure-linux-2---package-removal)
+CentOS 7 | x86_64 | [insiders-fast](#centos-7---insiders-fast-amd64) | [link](#centos---package-removal)
+CentOS 8 | x86_64 | [insiders-fast](#centos-8---insiders-fast-amd64) | [link](#centos---package-removal)
+Debian 10 | x86_64 | [insiders-fast](#debian-10---insiders-fast-amd64) / [prod](#debian-10---prod-amd64) | [link](#debian---package-removal)
+Debian 11 | x86_64 | [insiders-fast](#debian-11---insiders-fast-amd64) / [prod](#debian-11---prod-amd64) | [link](#debian---package-removal)
+Debian 12 | x86_64 | [insiders-fast](#debian-12---insiders-fast-amd64) | [link](#debian---package-removal)
+Red Hat Enterprise Linux 7 | x86_64 | [insiders-fast](#red-hat-enterprise-linux-7---insiders-fast-amd64) | [link](#red-hat-enterprise-linux---package-removal)
+Red Hat Enterprise Linux 8 | x86_64 | [insiders-fast](#red-hat-enterprise-linux-8---insiders-fast-amd64) | [link](#red-hat-enterprise-linux---package-removal)
+Red Hat Enterprise Linux 9 | x86_64 | [insiders-fast](#red-hat-enterprise-linux-9---insiders-fast-amd64) | [link](#red-hat-enterprise-linux---package-removal)
+Rocky Linux 9 | x86_64 | [insiders-fast](#rocky-linux-9---insiders-fast-amd64) | [link](#rocky-linux---package-removal)
+SUSE Linux Enterprise Server 15 | x86_64 | [insiders-fast](#suse-linux-enterprise-server-15---insiders-fast-amd64) | [link](#suse-linux-enterprise-server---package-removal)
+Ubuntu 20.04 | x86_64 | [insiders-fast](#ubuntu-2004---insiders-fast-amd64) / [prod](#ubuntu-2004---prod-amd64) | [link](#ubuntu---package-removal)
+Ubuntu 22.04 | x86_64 | [insiders-fast](#ubuntu-2204---insiders-fast-amd64) / [prod](#ubuntu-2204---prod-amd64) | [link](#ubuntu---package-removal)
+
+## Amazon Linux 9 - insiders-fast (AMD64)
+```bash
+yum install yum-utils
+yum-config-manager --add-repo https://packages.microsoft.com/yumrepos/microsoft-amazonlinux2-insiders-fast-prod/config.repo
+yum install osconfig
+```
+
+## Amazon Linux 9 - Package Removal
+```bash
+yum remove osconfig
+```
```

Comment:
Why not in a table rather than many sections?

Reply:
ill see if it looks better, standby

Reply:
Take it back. Sections looks good.